content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -146,7 +188,46 @@ Be aware that:
 ### Signature
 
 ```ts
-$executeRaw<T = any>(query: string | TemplateStringsArray, ...values: any[]): Promise<number>;
+$executeRaw<T = unknown>(query: TemplateStringsArray | Prisma.Sql, ...values: any[]): PrismaPromise<T>;
+```
+
+## <inlinecode>$executeRawUnsafe</inlinecode>
+
+The `$executeRawUnsafe` method allows you to pass a raw string (or template string) to the database. Like `$executeRaw`, it does **not** return database records, but returns the number of rows affected.
+
+<Admonition type="warning">
+
+By using this method you open up the possibility for SQL injection attacks. SQL injection attacks can expose your data, be it confidential or otherwise sensitive, to being deleted, modified, or even destroyed.
```

Comment:
```suggestion
By using this method you open up the possibility for SQL injection attacks. SQL injection attacks can expose your data, be it confidential or otherwise sensitive, to being modified, or even destroyed.
```